React/Modules/RCTDevSettings.mm

```diff
@@ -508,7 +508,12 @@ @implementation RCTBridge (RCTDevSettings)
 - (RCTDevSettings *)devSettings
 {
 #if RCT_DEV
-  return [self moduleForClass:[RCTDevSettings class]];
+  RCTDevSettings *devSettings = nil; // [TODO(OSS Candidate ISS#2710739)
+  Class klass = [RCTDevSettings class];
```

Comment:
"devSettingsClass" probably seems better?

Reply:
There are other instances in react-native using `klass`.  I not all of them made be me :)